### PR TITLE
feat: [P4PU-435] Call service logout

### PR DESCRIPTION
**Pull request by vito80ba**
This PR adds the call to the BE service "logout" 

#### List of Changes
- add call to the service logout

#### Motivation and Context
We' ve to notify the logout of a user calling the BE service and clearing the storage

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by AnisLucidi01 — DISMISSED** [review]

File: src/components/Header/index.tsx [hunk 1]
```diff
@@ -25,7 +25,17 @@ export const Header = (props: HeaderProps) => {
...
+    try {
+      await utils.apiClient.logout.getLogoutEndpoint();
+    } catch (e) {
+      console.warn(e);
+    } finally {
+      sessionClear(() => navigate(ArcRoutes.LOGIN));
+    }
+  }
+
+  const logout = () => logoutUser();
```
Comment (mmoio) [inline src/components/Header/index.tsx:38]: Can we remove this wrapping and using the logoutUser function directly?

**Review by mmoio — CHANGES_REQUESTED** [review]

File: src/components/Header/index.tsx (hunk 1 above)
Comment (stratoivandiluccio) [inline src/components/Header/index.tsx:38]: Why not declaring the the one on line 28 logout? I think you could remove the line 38 than.

Comment (vito80ba) [reply src/components/Header/index.tsx:38]: right

File: src/components/Header/index.tsx [hunk 2]
```diff
@@ -50,7 +58,7 @@ export const Header = (props: HeaderProps) => {
     {
       id: 'logout',
       label: 'Esci',
-      onClick: logout,
+      onClick: () => logoutUser(),
```
Comment (stratoivandiluccio) [inline src/components/Header/index.tsx]: please, just logoutUser. :)

Comment (mmoio) [inline src/components/Header/index.tsx:61]: Can we pass logoutUser without wrapping in an arrow function here?

**Review by mmoio — CHANGES_REQUESTED** [review]

**Review by mmoio — DISMISSED** [review]

**Review by stratoivandiluccio — APPROVED** [review]

**Pull request merged**